packages/pagination/src/usePagination.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { useSelection } from '@zendeskgarden/container-selection';
+
+/**
+ * A React Hook used to create accessible widgets with the Pagination Interaction Pattern
+ *
+ * https://www.w3.org/TR/wai-aria-practices/#Listbox
```

Comment:
If this is implementing the ARIA Listbox pattern, why don't we name this container `listbox` and implement accordingly? I'm a little confused between what I see implemented here and what the listbox pattern is calling for.

Reply:
Removed as that's not the case here